app/components/standup-deck.js

```diff
@@ -192,21 +259,31 @@ export default Ember.Component.extend(EKMixin, {
     return ("00" + String(this.get('elapsedMinutes'))).slice(-2);
   }),
   isEndingSoon: Ember.computed('elapsedMinutes', 'elapsedSeconds', function () {
-    if (this.get('state') === 'startups') {
-      return (this.get('elapsedMinutes') * 60 + this.get('elapsedSeconds')) > this.get('STARTUP_SLIDE_ENDS_SOON_AT');
-    } else {
-      return (this.get('elapsedMinutes') * 60 + this.get('elapsedSeconds')) > this.get('META_SLIDE_ENDS_SOON_AT');
+    switch(this.get('state')) {
+      case 'startups':
+        return (this.get('elapsedMinutes') * 60 + this.get('elapsedSeconds')) > this.get('STARTUP_SLIDE_ENDS_SOON_AT');
+      case 'incubators':
+        return (this.get('elapsedMinutes') * 60 + this.get('elapsedSeconds')) > this.get('INCUBATOR_SLIDE_ENDS_SOON_AT');
+      case 'meta':
+        return (this.get('elapsedMinutes') * 60 + this.get('elapsedSeconds')) > this.get('META_SLIDE_ENDS_SOON_AT');
```

Comment:
Tu peux sans doute extraire un `let elapsedSeconds = this.get('elapsedMinutes') * 60 + this.get('elapsedSeconds')` en haut de ce switch.